recipe/meta.yaml

```diff
@@ -1,29 +1,24 @@
-{% set version = "2021.6.0" %}
-{% set buildnumber = 1 %}
+{% set version = "2023.0.2" %}
+{% set buildnumber = 0 %}
 
 package:
   name: daal4py
   version: {{ version }}
 
 source:
   url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
-  sha256: a75acdc518f0af262a86952ee1040d5c8b9a9dbd4bf798fd974a917ee5b47e68
+  sha256: e289f49b34f2df3a3c5650c38a0d3080062169a9ba6e4bdbd154a4f0b982800a
   patches:
-    - patches/0001-replace-check_estimator_sparse_data-in-tests.patch
-    # For patch 0002, see: https://github.com/python/cpython/issues/84006
-    - patches/0002-win-64-py37-distutils.patch  # [win and py==37]
     - patches/0004_set_cmake_python_executable.patch
 
 build:
   number: {{ buildnumber }}
-  skip: True  # [not ((win or linux or osx) and x86_64)]
-  skip: True  # [py<36]
-  missing_dso_whitelist:      # [osx]
-    - lib/libc++.1.dylib      # [osx]
+  skip: True  # [not ((win or linux) and x86_64)]
+  skip: True  # [py<38]
```

Comment:
Seems like it has python 3.7 support too https://github.com/intel/scikit-learn-intelex/blob/ef80ea962b05e6172df600801d883c454db84ba1/setup.py#L424. Is there any reason to skip py<38?